contracts/glv/GlvToken.sol

```diff
@@ -0,0 +1,22 @@
+// SPDX-License-Identifier: BUSL-1.1
+
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import "../bank/Bank.sol";
+
+contract GlvToken is ERC20, Bank {
+    constructor(
+        RoleStore _roleStore,
+        DataStore _dataStore
+    ) ERC20("GMX Liquidity Vault", "GLV") Bank(_roleStore, _dataStore) {
```

Comment:
should we allow the GLV name and symbol to be customizable?

for GM we can't change it for existing GM tokens, but maybe for GLV it would be good to have custom names if there will be multiple GLV tokens